kernels/replace_intensities.cl

```diff
@@ -0,0 +1,17 @@
+const sampler_t sampler = CLK_NORMALIZED_COORDS_FALSE | CLK_ADDRESS_CLAMP_TO_EDGE | CLK_FILTER_NEAREST;
+
+__kernel void replace_intensities(
+    IMAGE_src0_TYPE  src0, 
+    IMAGE_src1_TYPE  src1,
+    IMAGE_dst_TYPE   dst
+)
+{
+  const int x = get_global_id(0);
+  const int y = get_global_id(1);
+  const int z = get_global_id(2);
+
+  const int index = (int) READ_IMAGE(src0, sampler, POS_src0_INSTANCE(x,y,z,0)).x;
```

Comment:
```suggestion
  const int index = convert_int(READ_IMAGE(src0, sampler, POS_src0_INSTANCE(x,y,z,0)).x);
```

Reply:
I am lost here, I though we wanted to avoid `convert_T` function for compatibility ?